### PR TITLE
open-cluster-management/backlog#4916 ts import cert

### PR DESCRIPTION
**Pull request by cadawson**

### CODE REVIEW

File: troubleshooting/trouble_clusters_offline_cert.adoc [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+== Symptom: All clusters offline after certificate change
+
+After you complete the procedure for updating a certificate secret, all of your clusters that were online are now displaying an `offline` status in the {product-title} console.
+
+[#identifying-all-clusters-offline-after-certificate-change]
+== Identifying the problem: All clusters offline after certificate change 
+
+After updating the information for a custom API server certificate, the clusters that were imported and running before the new certificate are now in an `offline` state. 
+
+The following errors are logged when the problem is the certificate change:
```
Comment (hanqiuzh) [inline troubleshooting/trouble_clusters_offline_cert.adoc]: we need to mention the logs are from pods in `open-cluster-management-agent ` namespace on a offline managed cluster

File: troubleshooting/trouble_clusters_offline_cert.adoc [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+----
++
+. Delete the outdated secret on the managed cluster to make sure the `registration-agent` uses the latest bootstrap secret to recreate secrets:
++
+----
+oc delete secret hub-kubeconfig-secret -n open-cluster-management-agent 
+----
++
+. Restart all pods in the `open-cluster-management-agent` namespace:
+    ```
```
Comment (hanqiuzh) [inline troubleshooting/trouble_clusters_offline_cert.adoc]: seems other code blocks are using `----`?

File: troubleshooting/trouble_clusters_offline_cert.adoc (hunk 1 above)
Comment (cadawson) [reply troubleshooting/trouble_clusters_offline_cert.adoc]: Added that.

File: troubleshooting/trouble_clusters_offline_cert.adoc (hunk 2 above)
Comment (cadawson) [reply troubleshooting/trouble_clusters_offline_cert.adoc]: Good catch. I missed that one. Fixed. Thanks!

**Review by hanqiuzh — APPROVED** [review]

**Comment by hanqiuzh** [comment]
~~@cadawson this PR is ready to merge to 2.0. 👍  and we will have enhancement for 2.1, so we will not need this doc for 2.1~~

**Comment by hanqiuzh** [comment]
Please hold this pr 😂  they updated the 2.0.z branch, I will double check if the steps can be simplified. Thanks.

**Comment by cadawson** [comment]
Ok. Was about to merge it, but will hold, for now. Thanks!

File: troubleshooting/trouble_clusters_offline_cert.adoc [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+----
+
+. Wait for 2-3 minutes for the cluster to connect, and for the `work-manager` to start.
+. Restart all pods in `open-cluster-management-agent-addon` namespace:
++
+----
+oc delete po —all -n open-cluster-management-agent-addon
+----
++
+The pods stop and use the new certificate information as they restart.
```
Comment (hanqiuzh) [inline troubleshooting/trouble_clusters_offline_cert.adoc:75]: all these steps

from line 56 
> Delete the outdated secret on the managed cluster to make sure the ...

to line 75
> The pods stop and use the new certificate information as they restart.

can be ignored/removed for 2.0.4 & 2.1.0

**Comment by hanqiuzh** [comment]
@cadawson the enhancement is merged for 2.0.4, and we will only need the reimport step for 2.0.4+

**Pull request merged**